zosia_site/services.tf

```diff
@@ -14,3 +14,13 @@ resource "google_project_service" "sql_service" {
   project = local.project_id
   service = "sql-component.googleapis.com"
 }
+
+resource "google_project_service" "artifactregistry_service" {
+  project = local.project_id
+  service = "artifactregistry.googleapis.com"
+}
+
+resource "google_project_service" "cloudrun_service" {
+  project = local.project_id
+  service = "run.googleapis.com"
+}
```

Comment:
You can change it to a single resource declaration with `for_each` and provide a list of services to activate.

Reply:
Done